swift3/test/functional/test_multi_upload.py

```diff
@@ -303,7 +304,7 @@ def test_object_multi_upload(self):
         self.assertTrue('etag' in headers)
         etag = headers['etag']
         self.assertTrue(etag.startswith('"'))
-        self.assertTrue(etag.endswith('-N"'))
+        self.assertTrue(etag.endswith('-1"'))
```

Comment:
It is "-1" because there is one part in the multipart object. The genuine swift3 does not compute neither the correct checksum nor the number of parts. I changed that in commit 012b666900e11633365551fa6b3bbed47746769b, but I only fixed half of the tests.